doc/user-guide.rst

```diff
@@ -199,134 +245,61 @@ Use one `$` for inline mode and `$$` for block mode.
 Miscellaneous
 """""""""""""
 
-You can insert a timestamp by clicking the clock icon or "Ctrl-Shift-d". You can also insert code snippets with highlighting, special characters, horizontal lines, etc...
+You can use basic text editor shortcuts and code snippets in the text editor to add highlighting, special characters, horizontal lines, etc...
+
+For a list of text shortcuts see this link: <https://www.tiny.cloud/docs/tinymce/latest/keyboard-shortcuts/>
+
+Examples:
 
+    ctrl+shift+d : add date/time at cursor
```

Comment:
make sure you compile the rst and look at the changes in html, then you can spot errors such as this:
![2024-12-18-165952_765x190_scrot](https://github.com/user-attachments/assets/3ffb822f-5921-441b-8805-efebca6f3fb1)

Reply:
fixed!